src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -61,6 +61,8 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
     bool m_is_chat_conversation;
     // InputsEmbedder
     std::shared_ptr<InputsEmbedder> m_inputs_embedder;
+    // Component for applying sampling to lm outputs
+    Sampler sampler;
```

Comment:
```suggestion
    Sampler m_sampler;
```
we typically name class fields with `m_` prefix

Also, `Sampler` has a method `clear_request_info` which drops sequence information once it's finished its execution.

I suppose we need to call this method once `generate` (non chat scenario) / `finish_chat` (finish chat) is called to release internal `Sampler` state for sequence info. Otherwise, consecutive `generate` will share `request_id = 0` and its state within sampler.

Reply:
changed to m_sampler 
added clear_request_info to finish chat and generate